templates/companies/companies_list.html

```diff
@@ -38,8 +40,7 @@ <h1>Companies</h1>
 	}
 </style>
 
-<form method="post" class="form_no_ul" enctype="multipart/form-data">
-	{% csrf_token %}
```

Comment:
Do we still have the `csrf_token` somewhere in the form? It is vital to preventing Cross-Site Request Forgery attacks, which is when a malicious site sends the user to this form with malicious payload, which immediately executes the form. The CSRF token is provided by the server in the initial GET request of the page, and the server won't process any form requests unless the CSRF token connected to the current session is valid, which prevents a form request from another page.

Since this form doesn't currently modify any data, it's not the worst thing but it is a good habit to have, in case the internal form handler ever changes behaviour.

Reply:
There is no code in `companies_list.html` that mentions the CSRF token, and when testing the site, the token is never in the URL. From this, I feel certain that the token is not in the form, but let me know if I've missed something!

Reply:
That means we never had a csrf token here anyway. Since we don't mutate data, I think it should be fine.